### PR TITLE
Updated to Hunting AWSCloudTrail and AuditLogs

### PR DESCRIPTION
**Pull request by juliango2100**
Fixes #

## Proposed Changes

  - Applies to AWS and AuditLog queries
  - Updated max time to 14 days to work with Hunting Portal
  - Updated entities 
  - Updated MITRE tactics

### CODE REVIEW

File: Hunting Queries/AWSCloudTrail/AWS_IAM_PolicyChange.txt [hunk 1]
```diff
@@ -28,5 +28,5 @@ AWSCloudTrail
 ,"CreatePolicy")
 | project TimeGenerated , EventName , EventTypeName  , UserIdentityAccountId ,  UserIdentityPrincipalid , UserAgent , UserIdentityUserName , SessionMfaAuthenticated , SourceIpAddress , AWSRegion , EventSource , AdditionalEventData , ResponseElements
 | extend IPCustomEntity = SourceIpAddress
-
+| extend HostCustomEntity = UserIdentityAccountId
```
Comment (timbMSFT) [inline Hunting Queries/AWSCloudTrail/AWS_IAM_PolicyChange.txt]: AccountCustomEntity ?

File: Hunting Queries/AzureActivity/Common_Deployed_Resources.txt [hunk 2]
```diff
@@ -6,7 +6,7 @@
 // in combination with other signals that show suspicious deployment to evaluate if the resource is one
 // that is commonly being deployed/created or unique.
 //
-// DataSource: #AzureActivity
+// DataSource: #AzureActivity, #Impact
```
Comment (timbMSFT) [inline Hunting Queries/AzureActivity/Common_Deployed_Resources.txt]: is the 'Impact intended to be in Tactics?

File: Hunting Queries/AzureActivity/Creating_Anomalous_Number_Of_Resources.txt [hunk 3]
```diff
@@ -5,7 +5,7 @@
 // Description: looks for anomalous number of resources creation or deployment activities in azure activity log.
 // It is best to run this query on a look back period which is at least 7 days.
 //
-// DataSource: #AzureActivity
+// DataSource: #AzureActivity, #Impact
```
Comment (timbMSFT) [inline Hunting Queries/AzureActivity/Creating_Anomalous_Number_Of_Resources.txt]: ditto - this change should be in Tactics line?

File: Hunting Queries/AWSCloudTrail/AWS_IAM_PolicyChange.txt (hunk 1 above)
Comment (juliango2100) [reply Hunting Queries/AWSCloudTrail/AWS_IAM_PolicyChange.txt]: fixed

File: Hunting Queries/AzureActivity/Common_Deployed_Resources.txt (hunk 2 above)
Comment (juliango2100) [reply Hunting Queries/AzureActivity/Common_Deployed_Resources.txt]: fixed

File: Hunting Queries/AzureActivity/Creating_Anomalous_Number_Of_Resources.txt (hunk 3 above)
Comment (juliango2100) [reply Hunting Queries/AzureActivity/Creating_Anomalous_Number_Of_Resources.txt]: fixed

**Pull request merged**